### PR TITLE
Grant Deployment Fixes

### PR DESCRIPTION
**Pull request by alexcelo**

### CODE REVIEW

File: deployment_0.json [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+        "canVote": true
+    },
+    {
+        "identifier": "0x18feebef2c9c08e0dbe1224ac1b167745477b3a7",
+        "beneficiary": "0x1286e16B0B963E7ED02BD23144da119f45aB3a94",
+        "releaseStartTime": "2018-12-10T08:00:00.000Z",
+        "releaseCliffTime": 3600,
+        "numReleasePeriods": 1,
+        "releasePeriod": 3600,
+        "amountReleasedPerPeriod": "134794.500000000000000000",
```
Comment (asaj) [inline deployment_0.json:41]: Why are we giving this out in an RG contract rather than a direct transfer?

File: deployment_0.json [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+        "subjectToLiquidityProvision": true,
+        "initialDistributionRatio": 1000,
+        "canValidate": false,
+        "canVote": true
+    },
+    {
+        "identifier": "0xbdfa9fd2237a95a67bb088a7a793519d4a17df33",
+        "beneficiary": "0xe3652a0353dCED174d3885E3091426e2d0969b38",
+        "releaseStartTime": "2020-05-20T07:00:00.000Z",
+        "releaseCliffTime": 2628000,
```
Comment (asaj) [inline deployment_0.json:182]: This one does not fit the typical patterns

File: deployment_0.json [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+        "canVote": true
+    },
+    {
+        "identifier": "0xd31ea05495967bca3fb89990415ea325b1e315cb",
+        "beneficiary": "0x328456453f390C44Bb3f851A339d344B0650a69a",
+        "releaseStartTime": "2020-06-01T07:00:00.000Z",
+        "releaseCliffTime": 31536000,
+        "numReleasePeriods": 1,
+        "releasePeriod": 3600,
+        "amountReleasedPerPeriod": "3050.000000000000000000",
```
Comment (asaj) [inline deployment_0.json:249]: Why so many with 3050?

File: deployment_4.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        "canVote": true
+    },
+    {
+        "identifier": "0x1c73c2181a4c6553cd192218f3be6c93dbd8859b",
+        "beneficiary": "0xd449DB2DAba9388266B78B5D40a4c40D72de36d0",
+        "releaseStartTime": "2020-06-05T07:00:00.000Z",
+        "releaseCliffTime": 31536000,
+        "numReleasePeriods": 1,
+        "releasePeriod": 3600,
+        "amountReleasedPerPeriod": "20000.000000000000000000",
```
Comment (asaj) [inline deployment_4.json:25]: If this is for running a validator group they will probably want 3-4 extra CELO

**Comment by alexcelo** [comment]
This is not for running a validator.

File: deployment_0.json (hunk 1 above)
Comment (alexcelo) [reply deployment_0.json:41]: I believe this was Marek's suggestion for this particular situation. I'll double check with him before deployment.

File: deployment_0.json (hunk 2 above)
Comment (alexcelo) [reply deployment_0.json:182]: Yeah, this one is on a 3 month deployment schedule with 1/3 vesting every 3 months.

File: deployment_0.json (hunk 3 above)
Comment (alexcelo) [reply deployment_0.json:249]: This is the net amount after taking out taxes, gross amount is 5k

File: deployment_4.json (hunk 4 above)
Comment (alexcelo) [reply deployment_4.json:25]: This is not for running a validator, but it's rather for a grant

Comment (alexcelo) [reply deployment_4.json:25]: A normal grant for completing a service milestone*

**Pull request merged**